gui/package.json

```diff
@@ -38,7 +38,8 @@
     "react-plotly.js": "^2.6.0",
     "react-router-dom": "^6.17.0",
     "react-visibility-sensor": "^5.1.1",
-    "tinystan": "^0.0.2"
+    "tinystan": "^0.0.2",
+    "vite-tsconfig-paths": "^4.3.2"
```

Comment:
Can this not be a dev- only dependency?

Reply:
Probably--I probably just installed it wrong. Let me change it and see if it still builds.